.github/workflows/ci.yaml

```diff
@@ -94,11 +95,16 @@ jobs:
             ci:
               - ".github/actions/**"
               - ".github/workflows/ci.yaml"
+            offlinedocs:
+              - "offlinedocs/**"
+
       - id: debug
         run: |
           echo "${{ toJSON(steps.filter )}}"
 
   lint:
+    needs: changes
+    if: needs.changes.outputs.offlinedocs-only == 'false' || needs.changes.outputs.ci == 'true' || github.ref == 'refs/heads/main'
```

Comment:
This will skip lint and format when we only target offlinedocs.
lint and fmt checks do not check offline docs so this was useless here